Makefile

```diff
@@ -3,28 +3,41 @@
 # Please refer to the LICENSE and AUTHORS files for details
 # SPDX-License-Identifier: MIT 
 
-CC := gcc
+CC     := gcc
+CFLAGS := -DPT_DEBUG -O2 -std=c99 -pedantic -Wall -Wextra
+
+LUA_DIR := /usr
+
 INSTALL_DIR    := /usr/local
 INSTALL_INCDIR := $(INSTALL_DIR)/include
 INSTALL_LIBDIR := $(INSTALL_DIR)/lib
+INSTALL_BINDIR := $(INSTALL_DIR)/bin
 
-.PHONY: install ptracer_header libptracer uninstall clean
+.PHONY: install ptracer_header pt-run libptracer uninstall clean
 
-install: ptracer_header libptracer
+install: ptracer_header pt-run libptracer
```

Comment:
This makefile needs some attention (#11).
Do you want to do it now or on the next PR?

Reply:
Let's do it in upcoming PRs